### PR TITLE
Prime-965 custom email

### PR DESCRIPTION
**Pull request by hogmann**

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/adjudication/shared/services/adjudication-resource.service.ts [hunk 1]
```diff
@@ -170,6 +170,21 @@ export class AdjudicationResource {
...
 
+  public logEmailInitiated(enrolleeId: number): NoContent {
+    return this.apiResource.post<NoContent>(`enrollees/${enrolleeId}/email-initiated`)
+      .pipe(
+        map(() => {
+          this.toastService.openErrorToast('Enrollee Email Initiated has been sent');
+        }),
+        catchError((error: any) => {
+          this.toastService.openErrorToast('Enrollee Email Initiated could not be logged');
+          this.logger.error('[Enrolment] EnrolmentResource::sendEnrolleeReminderEmail error has occurred: ', error);
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/shared/services/adjudication-resource.service.ts:181]: Comment should have `...::logEmailInitiated`

File: prime-angular-frontend/src/app/modules/adjudication/shared/services/adjudication-resource.service.ts [hunk 2]
```diff
@@ -170,6 +170,21 @@ export class AdjudicationResource {
       );
   }
 
+  public logEmailInitiated(enrolleeId: number): NoContent {
+    return this.apiResource.post<NoContent>(`enrollees/${enrolleeId}/email-initiated`)
+      .pipe(
+        map(() => {
+          this.toastService.openErrorToast('Enrollee Email Initiated has been sent');
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/shared/services/adjudication-resource.service.ts:177]: Success and error toast reads a bit funny.  Could it be `"Enrollee email event has been created"`, or something similar?

File: prime-angular-frontend/src/app/modules/adjudication/shared/services/adjudication-resource.service.ts [hunk 3]
```diff
@@ -170,6 +170,21 @@ export class AdjudicationResource {
...
+        }),
+        catchError((error: any) => {
+          this.toastService.openErrorToast('Enrollee Email Initiated could not be logged');
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/shared/services/adjudication-resource.service.ts:180]: Success and error toast reads a bit funny.  Could it be `"Enrollee email event could not be created"`, or something similar?

File: prime-angular-frontend/src/app/modules/adjudication/shared/services/adjudication-resource.service.ts [hunk 4]
```diff
@@ -170,6 +170,21 @@ export class AdjudicationResource {
       );
   }
 
+  public logEmailInitiated(enrolleeId: number): NoContent {
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/shared/services/adjudication-resource.service.ts:173]: Could this be `createInitiatedEnrolleeEmailEvent`?

File: prime-dotnet-webapi/Controllers/EnrolleesController.cs [hunk 5]
```diff
@@ -524,6 +524,35 @@ public async Task<ActionResult> sendEnrolleeReminderEmail(int enrolleeId)
...
+        /// </summary>
+        /// <param name="enrolleeId"></param>
+        [HttpPost("{enrolleeId}/email-initiated", Name = nameof(logEmailInitiated))]
+        [Authorize(Policy = AuthConstants.READONLY_ADMIN_POLICY)]
+        [ProducesResponseType(typeof(ApiBadRequestResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        public async Task<ActionResult> logEmailInitiated(int enrolleeId)
```
Comment (mtpultz) [inline prime-dotnet-webapi/Controllers/EnrolleesController.cs:539]: Could this be `createInitiatedEnrolleeEmailEvent`?

**Review by mtpultz — APPROVED** [review]
🍦 🥪

**Review by james-hollinger — APPROVED** [review]
LGTM

**Review by AnaisHebert — APPROVED** [review]
🎈

**Pull request merged**